src/main/resources/assets/create/lang/zh_cn.json

```
@@ -994,7 +1074,7 @@
 	"create.schematic.tool.flip.description.3": "",
 
 	"create.schematics.synchronizing": "正在同步..",
-	"create.schematics.uploadTooLarge": "你的蓝图太大",
+	"create.schematics.uploadTooLarge": "您的蓝图超出了服务器指定的限制。",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
	"create.schematics.uploadTooLarge": "你的蓝图超出了服务器指定的限制。",
```

---

这个有什么问题吗

---

一般而言不会用您。